nncf/api/compression.py

```diff
@@ -182,6 +182,22 @@ def scheduler(self) -> CompressionScheduler:
         :return: The instance of the `CompressionScheduler`.
         """
 
+    def load_state(self, state: Dict[str, object]) -> None:
```

Comment:
`load_state` should be an abstract method.

Reply:
Done